communication/marketing-team/role-handbooks/Events.md

```diff
@@ -0,0 +1,42 @@
+# Event Co-ordinator Role Handbook 
+
+## Overview
+
+As Event Co-ordinator Lead, you are responsible for
+- connecting with organizers of contributor events such as the monthly community meeting or contributor summits
+- working with the Contributor Comms team to ensure communications about events are being created and promoted through various means including email (k-dev), slack (slack bot, etc), twitter (K8sContributors), etc.
+
+Time Commitment:
+
+1-3 hours a week from 0-1.5 months in
+2-5 hours a week from 1.5 months-to event 
+
+## Minimum skills and Requirements
+
+- Kubernetes github org [membership](https://github.com/kubernetes/community/blob/master/community-membership.md)
+- familiarity with the workflow of [#sig-contributor-comms](https://github.com/kubernetes/community/tree/master/communication/marketing-team)
+- has understanding of the [social guidlines](https://github.com/kubernetes/community/blob/master/communication/marketing-team/storytelling-resources/social-guidelines.md)
+- Good grasp on general marketing activities surrounding events
+- Understanding of common forms of marketing communications
+- Event planning experience is a plus
+- Willing and able to directly coordinate across timezones and contributor groups
+- commitement for the role is relative and would vary depending upon events
+
+## Expectations and tasjs
+
+The core task for the role would be to stay current on [upcoming]() events. Planning should be done with the team about the event, target audience, mode of communication, timeline, etc. This could be coordinated via a call or async conversatios. Sometimes, some big events might require you to join a weekly or bi-weekly call. The following are the possible tasks to perform (may vary depending upon the event).
+
+- Creating a communication schedule - what, when, how, who
+- Promoting on different platforms like 
+    - emailing on [kdev](https://github.com/kubernetes/community/blob/master/communication/mailing-list-guidelines.md)
+    - twitting from main or k8scontributor accounts
+    - coordinating with CNCF(if required)
+    - sending slack notification to various channels(depending on the target audience)
+- Assisting in the entire blog process
+- Coordinating with Contributor Comms team members to ensure above tasks are completed
+
+Timeline for the communication can be crucial parameters so ideally timelines should at least begin to be decided weeks before the event. 
+
+## Shadowing the Event Coordinator Role
```

Comment:
Aside from shadowing, how does succession for this role work in general?

Reply:
Good point, Josh. I did add a bit more detail on how the role handoff works in the Social Media role handbook we added recently. Take a look at that, @AvineshTripathi and let's work together on adding a bit more detail here. Your suggestions are more than welcome, @jberkus!  Shadowing hasn't been a thing really in our group so we're trying to implement these for the first time really. https://github.com/kubernetes/community/blob/master/communication/marketing-team/role-handbooks/Social-Media.md

Reply:
If this person is going to be the contact point for all things events, I'd really like to see an aggressive and documented rotation schedule for the role.  That's the sort of role where you *have* to change out more than once a year.

Reply:
Ping!  I'd like to approve this but ... need there to be some kind of succession.